### PR TITLE
[irods/irods#6039] Added support for Rocky Linux 8 (main)

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -92,7 +92,7 @@ if (NOT CPACK_GENERATOR)
   elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "ubuntu" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "debian")
     set(CPACK_GENERATOR DEB CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
     message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
-  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux")
+  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky")
```
Comment (trel) [inline CMakeLists.txt]: consider... the IN_LIST notation?    might be useful here and in other places...

https://stackoverflow.com/a/23323790/6561189

Comment (korydraughn) [reply CMakeLists.txt]: nice!

File: CMakeLists.txt [hunk 2]
```diff
@@ -92,11 +92,14 @@ if (NOT CPACK_GENERATOR)
   elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "ubuntu" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "debian")
     set(CPACK_GENERATOR DEB CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
     message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
-  elseif(MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "centos" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "fedora" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "almalinux" OR MUNGEFS_LINUX_DISTRIBUTION_NAME STREQUAL "rocky")
-    set(CPACK_GENERATOR RPM CACHE STRING "CPack generator to use, e.g. {DEB, RPM, TGZ}." FORCE)
-    message(STATUS "Setting unspecified CPACK_GENERATOR to ${CPACK_GENERATOR}. This is the correct setting for normal builds.")
   else()
-    message(FATAL_ERROR "Failed to set CPACK_GENERATOR for unknown Linux Distribution: ${MUNGEFS_LINUX_DISTRIBUTION_NAME}")
+    set(MUNGEFS_SUPPORTED_LINUX_DISTRIBUTION_NAMES "centos" "opensuse" "fedora" "almalinux" "rocky")
```
Comment (trel) [inline CMakeLists.txt]: add RPM to the name...?

Comment (korydraughn) [reply CMakeLists.txt]: which one do you like?
- MUNGEFS_SUPPORTED_LINUX_DISTRIBUTION_NAMES_RPM
- MUNGEFS_SUPPORTED_RPM_LINUX_DISTRIBUTION_NAMES
- MUNGEFS_SUPPORTED_RPM_BASED_LINUX_DISTRIBUTION_NAMES
- MUNGEFS_RPM_BASED_LINUX_DISTRIBUTION_NAMES

Comment (korydraughn) [reply CMakeLists.txt]: going with this one: `MUNGEFS_SUPPORTED_RPM_BASED_LINUX_DISTRIBUTION_NAMES`

**Comment by trel** [comment]
why not keep the existing `elseif()` logic?

do we even need to specify the name of the IN_LIST? aka can we use it just 'in line'...

certainly if we're using it elsewhere...

**Comment by korydraughn** [comment]
i don't think we can specify it inline. the docs mention variable names.

**Comment by trel** [comment]
agreed

**Comment by korydraughn** [comment]
this will require another release or we'd have to document how to adjust versions.json for people interested in rocky linux.

added #

**Comment by trel** [comment]
okay, yes, we can bump/release again in the run up to 4.3.0.

**Pull request merged**